src/Avalonia.Native/AvaloniaNativePlatform.cs

```diff
@@ -167,25 +168,4 @@ public IWindowImpl CreateEmbeddableWindow()
             throw new NotImplementedException();
         }
     }
-
-    public class AvaloniaNativeMacOptions
```

Comment:
This is not used and also changing ShowInDock does not work after the app is visible anyway.